tests/integration/s2n_tls13_handshake_tests.py

```diff
@@ -47,7 +50,25 @@ def verify_hrr_random_data(server, client):
 
     return result
 
+def key_update_recv(server, client):
+  
+    result = Result()
+    result.status = Status.PASSED
+    openssl_msg = "Message:" + str(uuid.uuid4())
+    if client.args[0] == "openssl":
+        client.stdin.write(("k\n\n").encode("utf-8"))
+        client.stdin.flush()
+        time.sleep(1)
+        client.stdin.write((openssl_msg + "\n\n").encode("utf-8"))
+        client.stdin.flush()
+
+        # Openssl prints out KEYUPDATE to stderr when it has been sent.
+        if wait_for_output(server, openssl_msg, 100) and "KEYUPDATE\n" in get_error(client, 7):
```

Comment:
I thought you were able to verify the data after key update?

Reply:
This is verifying the data. openssl_msg is the data that is sent after the key update.

Reply:
👍 I read it wrong, thanks!